docs/_ext/nbexport.py

```diff
@@ -200,6 +230,12 @@ def visit_math_block(self, node):
         self.write_markdown("$$\n{}\n$$\n\n".format(node.astext().strip()))
         raise nodes.SkipNode
 
+    def visit_image(self, node):
+        if "nbattach" in node and node["nbattach"]:
+            filename = os.path.basename(node["uri"])
+            self.write_markdown(f"![{filename}](attachment:{filename})\n")
```

Comment:
I think you should add two linebreaks here since markdown wants two, otherwise it'll still wrap weirdly:

![image](https://user-images.githubusercontent.com/19836026/210254939-831c973d-4e56-4489-b306-f85254c10097.png)

```suggestion
            self.write_markdown(f"![{filename}](attachment:{filename})\n\n")
```